pkg/cmd/eventlistener/describe.go

```diff
@@ -95,8 +104,13 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
 {{ " " }}
 {{- end }}
 {{- end }}
+{{- if isTemplateRefExist $v.Template }}
+ TEMPLATE REF 
+ {{ decorate "bullet" $v.Template.Ref }}
```

Comment:
Should `APIVERSION` be added here as well for consistency?

Reply:
@danielhelfand Thanks for bringing this up, I too had a similar thought but was not sure if it was required but yes for consistency we can add it. I'll update the it.